imapautofiler/rules.py

```diff
@@ -254,6 +255,28 @@ def __init__(self, rule_data, cfg):
         super().__init__(rule_data, cfg)
 
 
+class TimeLimit(Rule):
+    """True if message is older than the specified 'age' measured
+     in number of days."""
+
+    _log = logging.getLogger('TimeLimit')
+    NAME = 'time-limit'
+
+    def __init__(self, rule_data, cfg, ):
+        super().__init__(rule_data, cfg)
+        self._age = rule_data['time-limit']['age']
+
+    def check(self, message):
+        date = parsedate_to_datetime(message['date'])
+
+        if self._age:
+            time_limit = datetime.now() - timedelta(days=self._age)
+            if date <= time_limit:
```

Comment:
Because the expression "date <= time_limit" results in a boolean, it can just be returned. We can fix that up in another patch.